cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/streaming/function/StreamingMultiSinkFunction.java

```diff
@@ -118,15 +124,7 @@ public void call(JavaRDD<RecordInfo<Object>> data, Time batchTime) throws Except
       MultiSinkFunction multiSinkFunction = new MultiSinkFunction(sec, phaseSpec, group, collectors);
       Set<String> outputNames = sinkFactory.writeCombinedRDD(data.flatMapToPair(multiSinkFunction),
                                                              sec, sinkNames);
-      sec.execute(new TxRunnable() {
```

Comment:
similar question here, why do we want to pass in a DatasetContext instead of doing it this way?

Reply:
Reverted.